README.md

```diff
@@ -15,17 +15,21 @@ Every spec has its own lifecycle that shows its maturity. We indicate this in a
 
 No stable specs right now. De facto a lot of the draft ones are stable and indicating this is work in progress.
 
+- [Status Client Specification](status-client-spec.md). The main specification for writing a Status client. **Start here**
+
 ### Draft
 
 The following SIPs are under consideration for standardization.
 
-- [Status Client Specification](status-client-spec.md). The main specification for writing a Status client. **Start here**
 - [Status Secure Transport Specification](status-secure-transport-spec.md). How Status provide a secure transport with conversational security properties.
 - [Status Payload Specification](status-payloads-spec.md). What the message payloads look like.
 - [Status Account Specification](status-account-spec.md). What a Status account is and how trust is established.
 - [Status Whisper Usage Specification](status-whisper-usage-spec.md). How we use Whisper to do routing, metadata protection and provide 1:1/group/public chat.
 - [Status Whisper Mailserver Specification](status-whisper-mailserver-spec.md). How we use Whisper mailservers to provide offline inboxing.
 - [Status EIPs Standards](status-EIPs.md). Ethereum Improvement Proposals used in Status.
+- [Status Blockchain Usage Specification](./status-blockchain-spec.md). How we use the Ethereum blockchain to facilitate the Web3 Browser and Wallet functionality
```

Comment:
These aren't draft, they are barely raw at this stage. I'd remove these and then we can include as draft once there is something here